### PR TITLE
OPSEXP-2240 Provide a rockylinux 9 + tomcat 10 tomcat base image

### PR DESCRIPTION
**Pull request by slohe1**
OPSEXP-2240

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -30,17 +30,26 @@ jobs:
...
           - flavor: rockylinux
             major: 8
+          - flavor: rockylinux
+            major: 9
         java_major:
           - 11
           - 17
         jdist:
           - jre
-          - jdk
```
Comment (gionn) [inline .github/workflows/main.yml:38]: remove jdist from here and dockerfile if we don't need it anymore

**Review by gionn — COMMENTED** [review]
there are a bunch of dependabots open on this repo, merge them and verify as part of this jira

**Review by alxgomz — CHANGES_REQUESTED** [review]
We should not produce this one as tomcat 10 is only supported starting with 23.1 and centos 7 is not anymore on the supported stack for 23.1.
the only centos7 based image we need is tomcat9 + jre 11

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -30,17 +30,28 @@ jobs:
...
+              major: 7
+            tomcat_major: 10
+          - base_image:
+              flavor: rockylinux
+              major: 9
+            tomcat_major: 9
+          - base_image:
+              flavor: rockylinux
+              major: 9
+            java_major: 11
```
Comment (gionn) [inline .github/workflows/main.yml:54]: at this point it's more concise if we just use `include`

**Review by gionn — APPROVED** [review]

**Review by alxgomz — APPROVED** [review]

**Pull request merged**